pjmedia/src/pjmedia/stream.c

```diff
@@ -1841,10 +1855,23 @@ static void handle_incoming_dtmf( pjmedia_stream *stream,
      * the DTMF digits in the buffer.
      */
     if (stream->dtmf_event_cb) {
+        /* Correct duration and timestamp for codecs where clock_rate
```

Comment:
since this is essentially the same code as above, should we put it in the outer block instead, to avoid duplication?

Reply:
yes, that sounds useful. I added a commit to reduce redundancy: 65f91a8b